common/djangoapps/third_party_auth/models.py

```diff
@@ -366,13 +366,12 @@ class OAuth2ProviderConfig(ProviderConfig):
 
     .. no_pii:
     """
-    # We are keying the provider config by backend_name here as suggested in the python social
-    # auth documentation. In order to reuse a backend for a second provider, a subclass can be
-    # created with seperate name.
+    # We are keying the provider config by backend_name and site_id to support configuration per site.
+    # In order to reuse a backend for a second provider, a subclass can be created with seperate name.
     # example:
     # class SecondOpenIDProvider(OpenIDAuth):
     #   name = "second-openId-provider"
-    KEY_FIELDS = ('backend_name',)
+    KEY_FIELDS = ('site_id', 'backend_name')
```

Comment:
I'm going to ask a Django-ignorant question, but just ensuring that there are no migrations required and that this doesn't actually affect the DB?

Reply:
No, it doesn't. This is a [class variable](https://github.com/openedx/django-config-models/blob/d2d04e5d3294d11ecdaf36a4474a3f7abd7d7d14/config_models/models.py#L80) in django-config-models.